slang.h

```diff
@@ -2098,6 +2098,21 @@ extern "C"
         SlangReflectionType* const* specializationArgs,
         ISlangBlob**                outDiagnostics);
 
+        /// Get the number of hashed strings
+    SLANG_API SlangUInt spReflection_getHashedStringCount(
+        SlangReflection*  reflection);
+
+        /// Get a hashed string. The number of chars is writtent in outCount. Note the count does *NOT* including terminating 0,
+        /// and the returned pointer will not generally have a terminating zero.
```

Comment:
It would be nice for the API contract here to guarantee that the return value has a null terminator, even if people are encouraged to rely on `outCount` in case there are embedded null bytes.

I'm guessing the reason for the caveat there has something to do with how the underlying storage that backs up this API is organized?